tapfreighter/chain_porter.go

```diff
@@ -171,22 +171,76 @@ func (p *ChainPorter) resumePendingParcels() error {
 		return err
 	}
 
+	if len(outboundParcels) == 0 {
+		log.Infof("Attempting to resume asset transfer for %d parcels",
```

Comment:
Was the `if len(...) == 0 {` check intentional here? Sounds like we might want to always log that?
Otherwise the log message might directly read "No pending parcels to resume"...